Mathlib/GroupTheory/Perm/Cycle/Concrete.lean

```diff
@@ -70,7 +70,7 @@ theorem formPerm_disjoint_iff (hl : Nodup l) (hl' : Nodup l') (hn : 2 ≤ l.leng
     all_goals have := formPerm_eq_self_of_not_mem _ _ ‹_›; tauto
 #align list.form_perm_disjoint_iff List.formPerm_disjoint_iff
 
-set_option linter.deprecated false in
+set_option linter.deprecated false in -- nthLe
 theorem isCycle_formPerm (hl : Nodup l) (hn : 2 ≤ l.length) : IsCycle (formPerm l) := by
```

Comment:
Is it hard to get rid of `nthLe` here?

Reply:
In fact, it wasn't; I pushed a commit doing so.